source/android/Renaissance/lib/http_services.dart

```diff
@@ -255,6 +255,58 @@ class HttpService {
     }
   }
 
+  Future<Tag> getTopic(String topic_id) async {
+    String url = baseUrl + '/topic/get-topic/$topic_id/';
+    Response res = await get(Uri.parse(url), headers: headers);
+
+    if (res.statusCode == 200) {
+      var temp = jsonDecode(res.body);
+      /*List<Milestonee> milestoness = [];
+      List<Topic> topicss = [];
+
+      (temp["milestones"]).map((tag) {
+        milestoness.add(Milestonee.fromJson(tag));
+      }).toList();
+
+      (temp["topics"]).map((tag) {
+        topicss.add(Topic.fromJson(tag));
+      }).toList();*/
+      return Tag(
+          id: topic_id,
+          description: temp['description'],
+          name: temp['name'],
+          isFav: temp['isFav']);
+    } else {
+      throw Exception(res.body);
+    }
+  }
+
+  Future<List<Tag>> getTopicList(String topic_id) async {
+    String url = baseUrl + '/topic/related-topic/$topic_id/';
+    Response res = await get(Uri.parse(url), headers: headers);
+    if (res.statusCode == 200) {
+      Iterable l = json.decode(res.body);
+      List<Tag> basicPaths =
```

Comment:
You should follow correct naming. Here topic list is stored in an array called basicPaths.

Reply:
Done